packages/notifications/resources/views/notification.blade.php

```diff
@@ -17,33 +25,47 @@
     ])"
     x-transition:leave-end="scale-95 opacity-0"
 >
-    @if ($icon = $getIcon())
-        <x-filament-notifications::icon :icon="$icon" :color="$getIconColor()" />
-    @endif
-
-    <div class="grid flex-1">
-        @if ($title = $getTitle())
-            <x-filament-notifications::title>
-                {{ str($title)->markdown()->sanitizeHtml()->toHtmlString() }}
-            </x-filament-notifications::title>
+    <div
```

Comment:
What's the purpose of the new wrapping div? (Possibly something with database notifications but I haven't checked the design locally.)

Reply:
The wrapping Div is to support the additional background color necessary on the base div so that the notifications aren't transparent. Also needed to compensate for the removal of the padding in the db notifications wrapper.

Reply:
Can't it be added to the root element instead?

Reply:
You can't have two background colors on the same element. One will override the other. So in order to have opacity for primary, etc. the wrapping Div still has to have the white and gray background.

Reply:
Didn't think about transparency, thanks.